src/main/resources/db/changelog/v.0.2/db.changelog-user.yaml

```diff
@@ -0,0 +1,54 @@
+databaseChangeLog:
+  - changeSet:
+      id: 03042021-add-table-user
+      author: maksim_maevsky
+      changes:
+        - createTable:
+            tableName: user
+            columns:
+              - column:
+                  name: user_id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_firstname
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_lastname
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_email
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_login
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_password
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_role_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: 04042021-add-foreignKeyConstraint-user-to-user-role
+      author:  maksim_maevsky
+      changes:
+        - addForeignKeyConstraint:
+            baseColumnNames: user_role_id
+            baseTableName: user
+            constraintName: fk_user_role
```

Comment:
See please naming conversion: 
https://www.c-sharpcorner.com/UploadFile/f0b2ed/what-is-naming-convention/#:~:text=The%20naming%20convention%20for%20a%20unique%20constraint%20is%3A,have%20a%20%E2%80%9CUQ_%E2%80%9D%20prefix.&text=If%20the%20unique%20key%20contains,separated%20with%20an%20underscore%20(_).
and 
https://stackoverflow.com/questions/199498/foreign-key-naming-scheme
If columns in tables have the same names, it's enough - `FK_<TargetTable>_<SourceTable>`
If totally different, it'll be better to specify

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/113558298-13248f80-9608-11eb-9bbc-c6d5a55785e2.png)

Reply:
@Maksim-maevsky if you've already  follow one naming style in DB, let's continue to follow it. So constraint will be fk_user_user_role
`FK_<TargetTable>_<SourceTable>`
Here : TargetTable is a table name that holds the Foreign Key and the Source Table is table name that hold the Primary Key.

TargetTable - user
SourceTable - user_role

Reply:
done